README.md

```diff
@@ -2,6 +2,10 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
+## Learning Objectives
+- Set up a React app.
+- Use React documentation.
+
 ## Available Scripts
 
 In the project directory, you can run:
```

Comment:
- [ ] Please use the [Microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md) instead of `create react app` default template and edit it with your project and personal data.

Reply:
Thank-you. doing that ASAP